### DISCUSSION TITLE
Is create_pool() function async?

### DISCUSSION THREAD

**Discussion by AptRoApt** [aio-libs/aiomysql#969] [Q&A]
I've looked at the sources and there are two functions: async _create_pool() and common create_pool(). Which one should I use?
The documentation uses "await create_pool()" but I can't understand why there is "await" keyword and is it asynchronous.

**Comment by bacchilu** [comment]
Not only this!

The `create_pool` function returns a `_PoolContextManager` instance instead of a `Pool` one.
